executor/include/execution/trait/common_traits.hpp

```diff
@@ -0,0 +1,12 @@
+//
+// Created by Shrijit Singh on 18/06/20.
+//
+
+#pragma once
+
+#define COMMON_TRAIT_NS execution
+
+namespace COMMON_TRAIT_NS {
```

Comment:
Feels like over-engineering.

Reply:
Agreed it is, one advantage is it just easier to integrate with PCL especially since the repo is changing so much. Once the repo is more stable will clean this all up.